odoo/models.py

```diff
@@ -2488,7 +2488,11 @@ def _auto_init(self):
                             cr.execute("SELECT indexname FROM pg_indexes WHERE indexname = %s and tablename = %s", (indexname, self._table))
                             res2 = cr.dictfetchall()
                             if not res2 and field.index:
-                                cr.execute('CREATE INDEX "%s_%s_index" ON "%s" ("%s")' % (self._table, name, self._table, name))
+                                try:  # DO NOT FORWARD-PORT TO SAAS-15 AND UP (stop at saas-14)
+                                    with cr.savepoint():
```

Comment:
the savepoint might not be necessary, since there are explicit commits all over the place
still i could not bring myself to not add it and hope for the best, i'd rather put safeguards in place